tests/azure/azure-pipelines.yml

```diff
@@ -39,6 +39,12 @@ stages:
     - script: pydocstyle .
       displayName: Verify docstings
 
+    - script: yamllint .
+      displayName: Verify docstings
```

Comment:
I don't think we need to keep lints in azure but if we do the displayName of this and the next step are mistaken.

Reply:
Removed all lints from Azure (which removed a whole stage).